sw/host/cryptotest/testvectors/parsers/nist_cavp_ecdsa_parser.py

```diff
@@ -15,35 +15,51 @@
 
 from cryptotest_util import parse_rsp, str_to_byte_array
 
+SUPPORTED_HASHES = [
+    "sha-256",
+    "sha-384",
+    "sha-512",
+    "sha3-256",
+    "sha3-384",
+    "sha3-512",
+]
 
 # Mapping from the curve names used by NIST to those used by cryptotest
 EC_NAME_MAPPING = {
     "P-256": "p256",
-    "P-384": "p384",
+#    TODO uncomment when cryptolib supports P-384
+#    "P-384": "p384",
```

Comment:
Soon: https://github.com/lowRISC/opentitan/pull/21067

Reply:
Nice. The comment was unfortunately necessary for now because NIST put the test vectors for all curves/hashes in a single file.